src/App.test.js

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
 
 import App from "./App";
+import store from "./redux/store";
```

Comment:
I think it would be best for uniformity purposes to use the mockStore that we setup as our store for testing redux. Some of your other tests are also using it. https://github.com/andela/ah-fortem-frontend/blob/d7bd0764dc351f128026a622794f5ee262f4fcdb/src/testutils/index.js#L20

Reply:
Changed